src/main/java/http/response/Response.java

```diff
@@ -46,4 +46,17 @@ public void setBody(byte[] body) {
     }
 
 
+
+    public void sendResponse () throws IOException {
+         if (this.body == null){  this.body = "".getBytes(); };
+        String formattedRequest =
+                "Content-Type: " + headers.get("Content-Type") + "\n"
+                + "Content-Length: " + body.toString().length() + "\n"
```

Comment:
these two headers are hardcoded - in the future, your responses might need other headers and it will become problematic. 

Since you're adding the headers that you want in handlers (via eg.  `response.addHeaders("Content-Type","text/html");`) , you should be able to use them dynamically when constructing the response in `sendResponse`.

Reply:
Yes I agree. This was to just get it working the build response class will handle this.